### DISCUSSION TITLE
Is routing name is case sensitive?

### DISCUSSION THREAD

**Discussion by Sandeep06Dev** [vercel/next.js#15539] [Help]
Hi, the intelligent guys.. 

Just want to know in next js the routing is case sensitive or not? 

If yes then how i can made in case insensitive. 

E.g.
I have a page "pages/myBio.jsx"

In browse when i type localhost:3000/myBio then pages gets open but when i type localhost:3000/MYBIO or localhost:3000/mybio 
in  browser address bar  then its not get open.. 

Help please.

**Comment by alanqchen** [comment]
The standard is that the path in a URL is case-sensitive. If you want to make case-insensitive paths work, I might look into [using custom routes](https://github.com/vercel/next.js/discussions/9081) and redirect everything to a lower-case path. 

You could also try using `getInitialProps` in `_error.js` and if there is an uppercase character, perform an 301 redirect.

But as a personal-opinion, unless it's critical that you're URL's are case-insensitive, I wouldn't stress about it.

**Reply by Sandeep06Dev** [reply]
Ok..thanks boss!! 
But The link page is too long.. Can you please provide any example??
Actually didn't know *"redirect everything in Lower case".. As for me M YBIO OR MyBio or mybio.. Should call same components..
Help!!
